### ISSUE TITLE
pip install --pre toga fails

### ISSUE THREAD

**Issue by diramazioni** [pybee/toga#562]
this is the output of   pip install --pre toga
```
Using cached https://files.pythonhosted.org/packages/c0/6b/a3ef1111bf1b841df9430c985a2c939ad491316c96af2369fc0863c3d859/toga-0.3.0.dev9.tar.gz
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-install-SU2tJ6/toga/setup.py", line 7, in <module>
        with io.open('src/core/toga/__init__.py', encoding='utf8') as version_file:
    IOError: [Errno 2] No such file or directory: 'src/core/toga/__init__.py'

```
I've also tried to pull directly the master branch of both toga and toga-gtk, toga-core setup installed toga-gtk that btw is version  0.2.4 and toga-core with no errors. ran python setup.py install on the toga master branch and this is the output
[toga-master_setup_install.txt](https://github.com/pybee/toga/files/2196998/toga-master_setup_install.txt)

So it got installed anyway beside the error but if I try to import toga this is the output:
```
Traceback (most recent call last):
  File "toga_demo/app.py", line 4, in <module>
    import toga
  File "/home/es/data/python/pybee/venv/bee3.6/lib/python3.6/site-packages/toga_core-0.2.4-py3.6.egg/toga/__init__.py", line 9, in <module>
    __version__ = '0.2.14'
  File "/home/es/data/python/pybee/venv/bee3.6/lib/python3.6/site-packages/toga_core-0.2.4-py3.6.egg/toga/interface/__init__.py", line 6, in <module>
    # __version__ = '1.2.3'       # Final Release
  File "/home/es/data/python/pybee/venv/bee3.6/lib/python3.6/site-packages/toga_core-0.2.4-py3.6.egg/toga/interface/widgets/box.py", line 1, in <module>
  File "/home/es/data/python/pybee/venv/bee3.6/lib/python3.6/site-packages/toga_core-0.2.4-py3.6.egg/toga/interface/widgets/base.py", line 1, in <module>
ImportError: cannot import name 'CSSNode'
```

**Comment by diramazioni** [comment]
Sorry I [wasn't following the guide](http://toga.readthedocs.io/en/latest/how-to/contribute.html) but now I' still have problem install toga-gtk I attach the output log of  pip install -e src/gtk
[pip_install_gtk.txt](https://github.com/pybee/toga/files/2197216/pip_install_gtk.txt)

**Comment by diramazioni** [comment]
I've solved this issue (in ubuntu 17.10) by installing these additional libraries:
libgirepository1.0-dev gobject-introspection python-dev libcairo2-dev python3-cairo-dev

I think the docs should be updated
